### PR TITLE
wasm: cleanup duplicate stats

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

Commit Message: Removes stat duplication for `wasm.v8.created` and `wasm_vm.v8.created` which appear to be measuring the same thing. Same for `active`.

Additional Description:
Risk Level: low
Testing:
Docs Changes: none since no docs written yet for wasm
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: source/extensions/common/wasm/wasm_vm.h [hunk 1]
```diff
@@ -18,36 +17,13 @@ namespace Extensions {
...
-                                    POOL_GAUGE_PREFIX(*scope_, runtime_prefix_))}) {
-    stats_.created_.inc();
-    stats_.active_.inc();
-    ENVOY_LOG(debug, "WasmVm created {} now active", runtime_, stats_.active_.value());
-  }
-  ~EnvoyWasmVmIntegration() override {
-    stats_.active_.dec();
-    ENVOY_LOG(debug, "~WasmVm {} {} remaining active", runtime_, stats_.active_.value());
-  }
+      : scope_(scope), runtime_(std::string(runtime)), short_runtime_(std::string(short_runtime)) {}
```
Comment (mathetake) [inline source/extensions/common/wasm/wasm_vm.h:26]: seems like we don't need to pass the scope here anymore?

Comment (mathetake) [reply source/extensions/common/wasm/wasm_vm.h:26]: same goes for other members?

Comment (kyessenov) [reply source/extensions/common/wasm/wasm_vm.h]: Thanks, deleted them except runtime which has a getter method and is used.

**Review by mathetake — APPROVED** [review]
LGTM, thank you!

**Review by lizan — APPROVED** [review]

**Pull request merged**